dc-image-processing/who.md

```diff
@@ -1,2 +1,2 @@
 The participants should:
-- Be familiar with Python and working with Bash
+- Be familiar with Python 
```

Comment:
I added only a bit, because I want to include people who have not used any tools or code for this yet, but want to. I'm OK with a wide range in Python, but I did add should be able to write simple functions (in case anyone thinks just pressing play on notebooks never coding is enough)